src/constants/language.js

```diff
@@ -41,6 +41,22 @@ export const successExportModal = {
   content: (email) => {
     return `An email has been sent to ${email}. This can sometimes take up to 15 minutes.`
   },
+  exportDataInfoHeader: 'Included in this export',
+  metadataExport: 'Metadata for this project',
+  sampleEventLevelExport: (dataSharing) => {
```

Comment:
Could you please separate the strings into individual tokens and keep functions out of the language file? At some point, these should move to a json file with just key-value tokens
```suggestion
  noSampleEventLevelData: 'No sample event level data',
  sampleEventLevelData: 'Sample event level data'
```

Reply:
I move the function to the SuccessExportModal and keep the text in the language as you suggest here